examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -73,9 +74,14 @@
     convert_unet_state_dict_to_peft,
     is_wandb_available,
 )
+from diffusers.utils.hub_utils import load_or_create_model_card, populate_model_card
```

Comment:
Going forward, prefer not to introduce multiple unrelated changes in a single PR. For example, this PR should primarily about B-LoRA training but here we're also refactoring the model card population related stuff.

Reply:
oh yeah that's actually not meant to go here, will remove that :)